jest.config.json

```diff
@@ -2,6 +2,7 @@
   "collectCoverage": true,
   "projects": [
     "<rootDir>/packages/facade/jest.config.json",
-    "<rootDir>/packages/common/jest.config.json"
+    "<rootDir>/packages/common/jest.config.json",
+    "<rootDir>/packages/config-nestjs/jest.config.json"
```

Comment:
All this is related to nestjs enterprise, so we could avoid nest prefix.
`packages/config-nestjs → packages/config`